bin/_test-helpers.sh

```diff
@@ -339,10 +339,12 @@ run_uninstall_test() {
 }
 
 run_deep_test() {
-  run_test "$test_directory/install_test.go" --multicluster
+  run_test "$test_directory/install_test.go"
```

Comment:
Tests are run with `--multicluster`  to test the installation (and uninstallation) of the multicluster components in `install_test.go`

Reply:
Ah yep didn't mean to remove this!